### PR TITLE
BudgetApp Ruby on Rails Project

### PR DESCRIPTION
**Pull request by taldr27**
### In this capstone project I:
- Created a Splash screen, sign-up, and login pages for authenticating a user.
- Created categories and transaction pages.
- Introduced new transactions associated with a category.
- Showed the money spent on each category.
- Added a "Add a new category" page.
- Added a "Add a new transaction" page.
- Added unit and integration tests.
image

To run the project locally kindly use: `bundle exec puma -C config/puma.rb`
[Loom Video Link](https://www.loom.com/share/63a2235bfa884ad9ba515776bdc364a8)
[Deployment on Render Link](https://budget-app-qrxi.onrender.com)
gif

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+### Key Features <a name="key-features"></a>
+- **Register and login features**
+- **Database where you can add categories and transactions**
+- **Implemented design from the sneak peek wireframe**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://budget-app-qrxi.onrender.com)
+
```
Comment (mwanawabangona) [inline README.md]: - [ ] `Required`
- [ ] Good readme file
- [ ] Good job on the live link 👍 
- [ ] You are also required to add a `Video presentation`  
1. Please recorded a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
4. The video should contain a demo of the project, highlighting all the project features.
5. The video should contain a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @taldr27 ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (taldr27) [reply README.md]: Thanks for your review @mwanawabangona , I forgot to add it to the `README.md` file

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @taldr27  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Descriptive pull request 📜 . I'm a fan of the alien 👽 by the way.
- Descriptive README 📜 
- Awesome adding the video presentation 📹 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by taldr27** [comment]
@Lucas-Masaba thank you for your review. 🍾

**Pull request merged**